aspect/src/main/java/tech/ydb/yoj/aspect/tx/YojTransactionAspect.java

```diff
@@ -83,12 +85,19 @@ private static String getSimpleMethod(Signature signature) {
         return signature.getDeclaringType().getSimpleName() + "." + signature.getName();
     }
 
-    Object safeCall(ProceedingJoinPoint pjp) {
+    CallResult safeCall(ProceedingJoinPoint pjp, Class<? extends Throwable>[] noRollbackExceptions) {
         try {
-            return pjp.proceed();
+            return CallResult.ofSuccess(pjp.proceed());
         } catch (RetryableException e) {
             throw new CallRetryableException(e);
         } catch (Throwable e) {
+            if (noRollbackExceptions != null) {
```

Comment:
Do we allow `RetryableException.class` and its subclasses in `noRollbackException`s? If no, we should add a validator; If yes, we should handle them in `catch (RetryableException)` block above, as well :-(